app/services/articles/feeds/weighted_query_strategy.rb

```diff
@@ -74,6 +74,7 @@ class WeightedQueryStrategy
       # - joins: [Optional] An SQL fragment that defines the join
       #          necessary to fulfill the clause of the scoring
       #          method.
+      # - disabled: [Optional] When true, we won't include this.
```

Comment:
Comment, non-blocking: I often find negated options a bit confusing. I wonder if it would be easier to use `enabled` with a default of `true` and change the options from `disabled: true` to `enabled: false`.

Reply:
I absolutely agree with the confusion of negated options.  In the moment I went with the quick `disabled: true` but didn't flip the defaults.  I will absolutely alter this, as it will make reading the logic easier.